xwiki-rendering-api/src/test/java/org/xwiki/rendering/listener/chaining/AbstractChainingListenerTest.java

```diff
@@ -44,7 +44,7 @@
  */
 class AbstractChainingListenerTest
 {
-    private static class AbstractChild extends AbstractChainingPrintRenderer
+    private static final class AbstractChild extends AbstractChainingPrintRenderer
```

Comment:
This class cannot be final as it's extended by other child classes, I did not notice when I merged it (I had hope that you tried to build it actually). Would be great if you could fix that @Kevin222004

Reply:
no issues let me study i have followed only regression report. and not build code in my local but give me time will study and fix it

Reply:
ohh.. thats was a huge mistake from my side. i have to update the code of checkstyle to fix this